pages/lab/api/container.md

```diff
@@ -26,6 +26,26 @@ import Container from '@material-ui/lab/Container';
 
 Any other properties supplied will be spread to the root element (native element).
 
+## CSS
+
+You can override all the class names injected by Material-UI thanks to the `classes` property.
+This property accepts the following keys:
+
+- `root`
+- `fixed`
+- `maxWidthXs`
+- `maxWidthSm`
+- `maxWidthMd`
+- `maxWidthLg`
+- `maxWidthXl`
+
+Have a look at [overriding with classes](/customization/overrides/#overriding-with-classes) section
```

Comment:
```suggestion
Have a look at the [overriding with classes](/customization/overrides/#overriding-with-classes) section
```

Reply:
Or maybe remove 'section' at the end so the sentence reads:

Have a look at overriding with classes and the implementation of the component for more detail.

Reply:
This is automatically generated, can we handle it in a different pull request? cc @mbrookes.

Reply:
Both work, so I guess shorter is better?